src/App.tsx

```diff
@@ -1,8 +1,105 @@
-import React from 'react';
-import './App.css';
+import React, { useState } from 'react';
+import { TodosList } from './TodosList';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+import { fullTaskList } from './api';
+import './App.scss';
+
+const App = () => {
+  const [todos, setTodos] = useState<FullTaskList>([]);
+  const [dataLoaded, setdataLoaded] = useState<boolean>(false);
+  const [loading, setLoading] = useState<boolean>(false);
+  const [button, switchButton] = useState<boolean>(true);
+
+  const getNewTasks = () => {
+    setLoading(true);
+    switchButton(false);
+    fullTaskList()
+      .then(data => {
+        setdataLoaded(!dataLoaded);
+        setLoading(false);
+        setTodos(data);
+      });
+  };
+
+  const sortBy = (action: string) => {
+    switch (action) {
+      case 'title':
+        setTodos(
+          [...todos].sort((a: { title: string }, b: { title: string }) => (
```

Comment:
this sorted list should be shown on the page and the new sorting should be done with the initial list, not the one you already sorted once